exampleSite/content/post/shortcodes.md

```diff
@@ -0,0 +1,21 @@
++++
+title = "Shortcodes"
+description = "A description of theme-specific shortcodes"
+date = "2021-06-21"
+author = "Hugo Authors"
++++
+
+## Sidenotes
+
+[LaTeX.css](https://latex.vercel.app/), which HugoTeX is using, defines syntax for sidenotes. However, as it is a little verbose to write, we provide a Hugo shortcode for that:
```

Comment:
I would appreciate it if you add the summary of this post like others.
Sorry you already know but, you can insert it as the summary contents and `<!--more-->` tags.

```
This article offers a sample of basic ...
<!--more-->

## Headings

The following HTML ...
```